test/api/model/web_auth_test.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:checks/checks.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:zulip/api/model/web_auth.dart';
+
+import '../../example_data.dart' as eg;
+
+void main() {
+  group('WebAuthPayload', () {
+    const otp = '186f6d085a5621ebaf1ccfc05033e8acba57dae03f061705ac1e58c402c30a31';
+    final encryptedApiKey = debugEncodeApiKey(eg.selfAccount.apiKey, otp);
+    final wellFormed = Uri.parse(
+      'zulip://login?otp_encrypted_api_key=$encryptedApiKey&email=self%40example&user_id=1&realm=https%3A%2F%2Fchat.example%2F');
```

Comment:
nit: break line for readability
```suggestion
    final wellFormed = Uri.parse(
      'zulip://login?otp_encrypted_api_key=$encryptedApiKey'
        '&email=self%40example&user_id=1&realm=https%3A%2F%2Fchat.example%2F');
```